### DISCUSSION TITLE
Grant user permission on folder level

### DISCUSSION THREAD

**Discussion by kevisong** [grafana/grafana#41802] [General]
Current behavior: User permission is org scope, user can either view, edit or operate as admin within the entire org.

Expected behavior: User can be granted view permission within one dashboard folder.

Use case: Currently, if user is granted viewer permission, this user can view all the dashboards within the org. Sometimes we just want the user to view dashboards within specific folder instead of all the dashboards in the org.

**Comment by ppolewicz** [comment]
The user will be able to run any query on the given database anyway, not just those queries that are written by the admins in the dashboards the user can access, so it might not matter much.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
